asghar/sprint1/infra/infra/infra_stack.py

```diff
@@ -0,0 +1,45 @@
+from aws_cdk import core as cdk
+from aws_cdk import aws_lambda as lambda_
+from aws_cdk import aws_iam
+from aws_cdk import aws_events as event_
```

Comment:
I was not keeping regular names because not to create conflit with already defined libraries, such as lambda is a function so it will create conflit if I define from aws_cdk import aws_lambda as lambda .

But if you say so I can change it.